st2common/st2common/metrics/drivers/prometheus.py

```diff
@@ -0,0 +1,48 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
```

Comment:
Maybe consistency in module naming? E.g. `statsd_driver` vs `prometheus`.

I assume you added `_driver` suffix because statsd name clashes 3rd party library. I'm fine going either way - adding `_driver` suffix to all the driver modules, or drop it everywhere and use `from __future__ import absolute_import` to avoid name clashes.

Reply:
Agree.